default/scripting/ship_parts/Troops/GT_TROOP_POD_2.focs.txt

```diff
@@ -9,6 +9,7 @@ Part
     tags = [ "PEDIA_PC_TROOPS" ]
     location = And [
         OwnedBy empire = Source.Owner
+        # @content_tag{NO_ATTACKTROOPS} Prevents construction for this part at a location with tag
```

Comment:
Prevents production of ships with this part in their design at locations with this tag